src/main/java/com/shopify/store/FruitProduct.java

```diff
@@ -0,0 +1,60 @@
+package com.shopify.store;
+
+
+/**
+ * FruitProduct Products
+ */
+public class FruitProduct implements Product{
+    private ProductType productType;
+    private String fruitName;
+    private long price;
+    private int quantity;
+
+    /**
+     *
+     * @param fruitName takes in the fruitName and in the future would implement unique SKU
+     * @param quantity how many of the fruit product
+     */
+    public FruitProduct (String fruitName, int quantity) {
+        this.fruitName = fruitName.toLowerCase();
+        this.quantity = quantity;
+        setPrice(fruitName.toLowerCase());
+        setProductType(ProductType.FRUIT);
+    }
+
+    public void setPrice(String fruitName) {
```

Comment:
`this` already has a `fruitName`

Reply:
Yeah i was contemplating on utilizing an ENUM for different categories of products, FRUIT being one of them for future add ons. I'll get rid of it

Reply:
Although I think enums are greatly underused and appreciated, you shouldn't use them for data that's rapidly changing that you'd want to load from a web service or config file at runtime.